src/web/lib/braze/FakeAppBoy.ts

```diff
@@ -0,0 +1,21 @@
+import { createNanoEvents, Emitter } from 'nanoevents';
```

Comment:
Now that we're only using this in one place, I almost wonder if it would be better defined within the test where it's used. Mostly just to make it super clear that it's a testing utility. Any thoughts on that?

Reply:
Good point - I've moved it into the test.